src/components/NavigationContainer.js

```diff
@@ -10,4 +10,4 @@ function mapStateToProps (state, ownProps) {
   }
 }
 
-export default connect(mapStateToProps)(Navigation)
+export default connect(mapStateToProps, null, null, { pure: false })(Navigation)
```

Comment:
This is part of the reason I'm a fan of programming languages that force you to name your arguments.